### DISCUSSION TITLE
Generating environment.yml

### DISCUSSION THREAD

**Discussion by akhmerov** [prefix-dev/pixi#1378] [Q&A] [answered]
Is there a way to use pixi to generate environment.yml? This may be useful for example if I need interoperability with tools that don't (yet) support pixi configuration. I see there's `pixi list --json`, but that doesn't quite cut it.

**Comment by pavelzw** [comment]
https://github.com/prefix-dev/pixi/issues/1216

**Reply by akhmerov** [reply]
Thanks! This isn't exactly the same as my question though: I am more wondering about generating the yaml that conda/mamba and other tools will understand.

**Reply by ruben-arts** [reply, marked as answer]
In the issue there is a conversation going on about `pixi export` with mutliple formats. I assume pavel was talking about that. 

There is not such a feature yet.

**Reply by ruben-arts** [reply]
Great work on this is done in #1427  by @abkfenris

**Comment by lucascolley** [comment]
https://pixi.sh/latest/reference/cli/pixi/workspace/export/conda-environment/
